bcdaworker/queueing/manager/que.go

```diff
@@ -107,6 +110,24 @@ func (q *queue) processJob(job *que.Job) error {
 		return nil
 	}
 
+	// start a goroutine that will periodically check the status of the parent job
+	go func() {
+		for {
+			select {
+			case <-time.After(15 * time.Second):
```

Comment:
How long does a typical job take? Is there a chance that a cancel is requested, but the job is completed anyway because it took less than 15 seconds?

I think this is fine for now, since we're aiming to cancel more time consuming jobs. Maybe we can put in a ticket to propagate the cancel at an earlier time -- like from the web package -- and get rid of the timer?